topic_folders/policies/immediate-response-checklist.md

```diff
@@ -0,0 +1,74 @@
+## Carpentries Staff Immediate Response Checklist
+
+When a Code of Conduct (CoC) violation is found, one course of action is that activities in Carpentries channels (in-person and virtual) are monitored. In other cases, the community member is asked to take a termed suspension. When this is the resolution of the CoC committee, the following boundaries are enforced. The checklist below was created for incidents that require an immediate response, i.e. the community member is removed from all Carpentries engagement.
+
+The intended outcomes for such a suspension are immediate, ongoing, and demonstrable changes in any areas of engagement for which the CoC deemed undesirable in the incident review. 
+
+### Online Communication and Communities
+Revoke access from the following communication channels and communities: 
+- [ ] Email lists on TopicBox 
+- [ ] Email lists on Google Groups
+  - [ ] Library Carpentry Governance Group
+  - [ ] Library Carpentry Maintainer Onboarding
+  - [ ] Code of Conduct Committee
+  - [ ] Carpentries Mentoring Groups 
+- [ ] Twitter - (@datacarpentry, @swcarpentry, @libcarpentry, @thecarpentries)
+- [ ] Blogs - (Data Carpentry, Software Carpentry, Library Carpentry, The Carpentries)
+- [ ] GitHub (carpentries, datacarpentry, swcarpentry, librarycarpentry, data-lessons, carpentrieslab)
+- [ ] Gitter - Library Carpentry Lobby
+- [ ] [Slack](https://swcarpentry.slack.com/messages)  
+- [ ] Facebook 
+- [ ] LinkedIn 
+
+Additionally, the community member will not be allowed to attend community discussions.
+
+### Teaching Workshops
+The suspended member will, for the duration of their suspension:
+- [ ] Be asked not to sign up for workshops on the Instructor Spreadsheet.
+- [ ] Not be able to participate as a helper at any workshops.
+- [ ] Be removed from Eventbrite login access.
+- [ ] Not be allowed to plan self-organized workshops.
+- [ ] Not be allowed to request or host centrally coordinated workshops.
```

Comment:
The two above might alert others to the fact that they are under review. However, having people potentially run workshops while under review is also not good. Maybe this has to be adjusted to the seriousness of the situation? I know this feels fuzzy, but still.

Reply:
Revoking all communication access during review should be an option depending on severity, but not a standard procedure for all incidents.  See my previous comment about active harassment being a reasonable situation for why this might happen, but my impression is that most incidents are one-offs.  Also managing these permissions is a huge load for staff, so I'd like to avoid this sort of response to every review if that's possible.

Reply:
I updated the intro paragraph and that language resolves this comment.